projects/observability/src/pages/custom-dashboards/custom-dashboards.component.ts

```
@@ -0,0 +1,25 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { EMPTY, Observable, of } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+import { CustomDashboardService, DashboardListItem } from './custom-dashboard.service';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  styleUrls: ['./custom-dashboards.component.scss'],
+  template: `
+    <div class="custom-dashboards">
+      <div class="dashboard-list" *htLoadAsync="this.dashboards$ as dashboards">
+        <ht-link class="dashboard" *ngFor="let dashboard of dashboards" [paramsOrUrl]="dashboard.link">
+          <h4>{{ dashboard.display }}</h4>
+        </ht-link>
+      </div>
+    </div>
+  `
+})
+export class CustomDashboardListComponent implements OnInit {
+  public dashboards$: Observable<DashboardListItem[]> = of([]);
+  public constructor(private readonly customDashboardService: CustomDashboardService) {}
+  public ngOnInit(): void {
+    this.dashboards$ = this.customDashboardService.fetchDashboards().pipe(catchError(() => EMPTY));
```

Why are we ignoring the error here?

---

Fixing